test/integration/targets/async/tasks/main.yml

```diff
@@ -75,7 +75,7 @@
 - name: assert task was successfully started
   assert:
     that:
-    - fnf_task.started
+    - fnf_task.started == 1
```

Comment:
this makes me want a 'started' test ....

Reply:
Yup, seems the right thing to do: #43445